eox_nelp/admin/student.py

```diff
@@ -30,10 +31,17 @@ def pearson_real_time_action(modeladmin, request, queryset):  # pylint: disable=
         queryset: The QuerySet of selected CourseEnrollment instances.
     """
     for course_enrollment in queryset:
-        real_time_import_task.delay(
-            course_id=str(course_enrollment.course_id),
-            user_id=course_enrollment.user.id,
-        )
+        if getattr(settings, "USE_PEARSON_ENGINE_SERVICE", False):
```

Comment:
I believe there are more flexible ways to handle task selection, avoiding the need to specify everything in every part. We could simply switch functions and pass additional arguments (kwargs) as needed. Let's continue because logic works.